reference/docs-conceptual/community/contributing/get-started-writing.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Get started contributing to PowerShell documentation
+description: This article is an overview of how to get started as a contributor to the PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Get started contributing to PowerShell documentation
+
+## PowerShell-Docs structure
+
+The [PowerShell-Docs repository][psdocs] is divided into two groups of content. Git branches are
+used to manage how and when documentation gets published.
+
+### Reference content
+
+The reference content is the PowerShell cmdlet reference for the cmdlets that ship in PowerShell.
+The [reference][ref] is collected in versions folders (5.1, 6, 7.0, and 7.1). This content contains
+cmdlet reference only for the modules that ship with PowerShell. This content is also used to create
+the help information displayed by the `Get-Help` cmdlet.
+
+> [!NOTE]
+> The Table of Contents (TOC) for reference content is autogenerated by the publishing system. You
+> don't have to update the TOC.
+
+### Conceptual content
+
+The conceptual documentation includes the following content:
+
+- Release notes
+- Setup instructions
+- Sample scripts and how-to articles
+- DSC documentation
+- SDK documentation
+
+The [conceptual documentation][conceptual] is not organized by version. All articles are displayed
+for every version of PowerShell. We are working to create version-specific articles. When that
+feature is available in our documentation, this guide will be update with the appropriate
+information.
+
+> [!NOTE]
+> Anytime a conceptual article is added, removed, or renamed, the TOC must be updated and included
```

Comment:
Can we provide a link for what that TOC update looks like?

Reply:
link to public doc when available